### ISSUE TITLE
27398707: There should be a way to know what has changed in a title when iBooks provides updates

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#15394]
#### Description

Summary:
When iBooks for Mac indicates that there is an update to an existing e-book, it is impossible to know what changed, and whether it is worth re-reading all or part of that item.

Steps to Reproduce:
1. Click on the Download item _when there are items pending update_.

Expected Results:
I would expect some link to release notes for that change that indicate which part of the book has changed, whether it is just metadata, images, or text. Also, there is no way to know if one's notes have been affected by the change.

I would further expect a tool to compare the new version with the current version.

It would also be possible, in case of doing references, to get the exact version for which a note was made.

Actual Results:
Just an update button that, given the lack of knowledge of what the update entails, should not exist, and have updates happen automatically.

Version:
iBooks for Mac 1.5
Mac OS X 10.11.5
## 

Product Version: 1.5
Created: 2016-07-18T08:54:37.018960
Originated: 2016-07-18T00:00:00
Open Radar Link: http://www.openradar.me/27398707
